dotnet/samples/README.md

```diff
@@ -0,0 +1,10 @@
+## Kernel Samples
+
+| Type                                                                        | Description                                                                                              |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| [`Getting started`](./GettingStarted/README.md)                             | A single step-by-step tutorial to get started                                                            |
+| [`Tutorials`](./Tutorials/README.md)                                        | More in depth step-by-step tutorials                                                                     |
+| [`Concepts`](./Concepts/README.md)                                          | Concepts by feature specific code snippets                                                               |
+| [`Demos`](./Demos/README.md)                                                | Demonstration applications that leverage the usage of one or many SK features                            |
```

Comment:
`Demonstration applications which show you how to use features of the Semantic Kernel`

I would avoid using "SK"